server.js

```diff
@@ -4,5 +4,12 @@ const {DashServer} = require('./build/server/dash-server');
 const {GitHub} = require('./build/utils/github');
 const secrets = require('./secrets.json');
 
+const logError = (err) => {
+  console.error(err);
+  throw err;
+};
+process.on('uncaughtException', logError);
+process.on('unhandledRejection', logError);
```

Comment:
Can we move this to the end of the file?

Reply:
done